canvas/animation.go

```diff
@@ -0,0 +1,60 @@
+package canvas
+
+import (
+	"image/color"
+	"time"
+
+	"fyne.io/fyne"
+)
+
+// DurationStandard is the time a standard interface animation will run.
+const DurationStandard = time.Millisecond * 300
+
+// NewColorAnimation sets up a new animation that will transition from the start to stop Color over
+// the specified Duration. The content of fn should apply the color values to an object and refresh it.
+// You should call Start() on the returned animation to start it.
+func NewColorAnimation(start, stop color.Color, d time.Duration, fn func(color.Color)) *fyne.Animation {
```

Comment:
Animating through the RGBA colour space gives a different result than through the HSLA colour space.
```suggestion
func NewRGBAColorAnimation(start, stop color.Color, d time.Duration, fn func(color.Color)) *fyne.Animation {
```

Reply:
Good catch thanks. However I'm not sure about the name. How would you feel about `NewColorAnimationRGB` or  `NewColorRGBAnimation`? (I think we can omit the `A` as that is not part of the colour model).

Reply:
I think `A` should be kept to be consistent with Color Picker and image.RGBA. `NewRGBAAnimation` & `NewHSLAAnimation` would be fine with me

Reply:
The point of putting `Color` earlier in the name as I illustrated was to make it more searchable and the methods would be grouped in ordering. Was your providing of two different names objecting to the suggestion?

Reply:
As per the call let's go with `NewColorRGBAAnimation` & `NewColorHSLAAnimation`